uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/apis/absence-type.api.spec.js

```diff
@@ -46,7 +46,8 @@ define([
       });
 
       it('sends GET request with correct parameters', function () {
```

Comment:
it would be nice to add a test for overriding `is_active` and `sort` parameters.

Reply:
Done.